### ISSUE TITLE
Sort Session

### ISSUE THREAD

**Issue by Surajit0573** [Aditya61101/Group-Study#8]
Currently, sessions can be sorted according to starting date.
I want to implement a dropdown menu functionality within a web application to sort a set of course data. The specific goal is to allow users to select between different sorting criteria like: "Starting Date" and "Maximum Students".

**Comment by Aditya61101** [comment]
> Currently, sessions can be sorted according to starting date.
> I want to implement a dropdown menu functionality within a web application to sort a set of course data. The specific goal is to allow users to select between different sorting criteria like: "Starting Date" and "Maximum Students".

The session can be sorted based on starting date, ending date, starting time and ending time as well. If sessions have same starting date then it will sort based on ending date and so on. But if you want to implement a dropdown to give user choices to sort sessions, you can go ahead and do it.

**Comment by Surajit0573** [comment]
Okay, I can implement the dropdown option for ease of user and sort according to their need. Please assign me under Codepeak!!

**Comment by Aditya61101** [comment]
> Okay, I can implement the dropdown option for ease of user and sort according to their need. Please assign me under Codepeak!!

Yup assigning you, but first go through contribution.md file to avoid any conflict.
